File: AntiClimacus/DataAccessLayer/Entities/Address.cs
using System.Collections.Generic;

namespace DataAccessLayer.Entities
{
    public class Address
    {
        public int Id { get; set; }
        public string City { get; set; }
        public string County { get; set; }
        public string Country { get; set; }
        public string Street { get; set; }
        public int Number { get; set; }

        public ICollection<Donor> Users{ get; set; }
    }
}

File: DonatorSange-Angular/src/app/models/address.ts
export class Address{
    id:number;
    city:string;
    country:string;
    county:string;
    number:number;
    street:string;
    static initialize(){
        let address = new Address();
        address.id = 0;
        address.city = null;
        address.country = 'Romania';
        address.county = null;
        address.number = 0;
        address.street = null;
        return address;
    }
}